packages/back-end/types/fact-table.d.ts

```diff
@@ -38,6 +41,7 @@ export interface ColumnInterface {
   alwaysInlineFilter?: boolean;
   topValues?: string[];
   topValuesDate?: Date;
+  jsonFields?: JSONColumnFields;
```

Comment:
nit:
Should `JSONColumnField = Record<string, {dataType: FactTableColumnType}>`? I find the naked use of returning the datatype in various functions like `getColumnExpression` to be a bit confusing. Or the fact that this is a map of datatypes could be clearer in the name.

Reply:
yeah, good call. I can see adding more info as well in the future, so an object is better